core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -64,7 +64,7 @@ private[spark] class Executor(
 
   private val EMPTY_BYTE_BUFFER = ByteBuffer.wrap(new Array[Byte](0))
 
-  private val conf = env.conf
+  protected val conf = env.conf
```

Comment:
We are calling the fetchFile() method based on the jars in the job local variables ( A thread local which is available accross all partitions of a job). fetchFile() needs this value.